operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -16,6 +16,7 @@
     private void write(File keyFile, String str) throws IOException {
         try (FileWriter writer = new FileWriter(keyFile)) {
             writer.write(str);
+            writer.close();
```

Comment:
This should be unnecessary since `writer` is the subject of a try-with-resources block, no?

Reply:
Ah, you are right.